src/main/java/address/ui/Ui.java

```diff
@@ -0,0 +1,34 @@
+package address.ui;
+
+import address.MainApp;
+import address.browser.BrowserManager;
+import address.controller.MainController;
+import address.model.ModelManager;
+import address.util.Config;
+import javafx.scene.control.Alert;
+import javafx.stage.Stage;
+
+/**
+ * The UI of the app.
+ */
+public class Ui {
+    MainController mainController;
+
+    public Ui(MainApp mainApp, ModelManager modelManager, Config config){
+        BrowserManager.initializeJxBrowserEnvironment();
```

Comment:
I only tried the initialization of jxBrowser, which only works in `MainApp` `init` method so what has been suggested actually is possible. Not sure about WebView as I was not involved.